subsystem/subsystem_suite_test.go

```diff
@@ -37,6 +37,7 @@ var Options struct {
 	TestTokenUnallowed   string `envconfig:"TEST_TOKEN_UNALLOWED"`
 	OCMHost              string `envconfig:"OCM_HOST"`
 	DeployTarget         string `envconfig:"DEPLOY_TARGET" default:"k8s"`
+	S3Mode               string `envconfig:"S3_MODE" default:"aws"`
```

Comment:
Related to my other comment about naming, would it make sense to call this `Storage` or `Storage` backend?

I'm more concerned about the options exposed (which will be harder to change without breaking backward compatibility) than what I am with the internal naming. Although, having everything aligned would be nice.

Reply:
Storage might be good with the options of "s3" and "filesystem"? I'm terrible at naming :)

Reply:
> Storage might be good with the options of "s3" and "filesystem"? I'm terrible at naming :)

lol, you and me both. I'd say, yeah. In past projects, I've used (and seen people using) terms like Storage, Backend, or Bucket* to categorize the various solutions supported to store these objects.